### ISSUE TITLE
Cannot use dfti functions with windows-x64 runtime

### ISSUE THREAD

**Issue by alexbird1985** [MKL-NET/MKL.NET#18]
Hi, I am having the following issue where one of the DLLs is not being built.

Using MKL.NET with the following versions:

MKL.NET 1.4.0
MKL.NET.Matrix 1.0.0
MKL.NET.win-x64 2022.0.0.115

I've been able to do some matrix and vector operations with no worries, but calling in to the C API does not work.

```
System.DllNotFoundException: Unable to load DLL 'MKL.NET.Native.dll' or one of its dependencies: The specified module could not be found. (0x8007007E)
   at MKLNET.Dfti.ComputeForwardInplace(Int32 n, Complex[] x_inout)
   at MKLNET.Dfti.ComputeForward(Complex[] x_inout)
   at Filtering.Fft.Resample(Double[] xs, Int32 num)
```

I have tried building Debug & Release for x64 & any-cpu targets.

I'm sorry if this turns out to be a stupid question, I just can't find any documentation for how to fix it myself.

Thanks.

**Comment by AnthonyLloyd** [comment]
Hi, What does your project file look like? Is it new style SDK with a RuntimeIdentifier specified?

**Comment by alexbird1985** [comment]
Hi,
This is the contents of my csproj:
```
<Project Sdk="Microsoft.NET.Sdk">

    <PropertyGroup>
        <OutputType>Exe</OutputType>
        <TargetFramework>net7.0</TargetFramework>
        <ImplicitUsings>enable</ImplicitUsings>
        <Nullable>enable</Nullable>
        <RuntimeIdentifier>win-x64</RuntimeIdentifier>
    </PropertyGroup>

</Project>
```
In other news, I had a look in the nuget package download and I can't see the MKL.NET.Native.DLL anywhere. I can see it is being built in CI but it doesn't seem to appear in the package itself?

![image](https://user-images.githubusercontent.com/52898676/230721856-4db547d5-69c5-45c2-91b9-00c3783ed5ea.png)
and
![image](https://user-images.githubusercontent.com/52898676/230721885-402fbb37-89f8-424e-b784-f50b9ce66059.png)

So I think it not loading might be because it simply isn't present? I can't find it as a build artifact either.

Thanks.

**Comment by alexbird1985** [comment]
OK, so I had a bit more of a look and I did find the DLL in the x86 runtimes folder of the main MKL.NET package.
I was able to get the project to run if I switch to the win-x86 runtime identifier, so it seems like the issue is that I am targeting x64.

Is this intended behaviour? I would like to be able to build this project as x64 if possible.

Updated csproj:
```
<Project Sdk="Microsoft.NET.Sdk">

    <PropertyGroup>
        <OutputType>Exe</OutputType>
        <TargetFramework>net7.0</TargetFramework>
        <ImplicitUsings>enable</ImplicitUsings>
        <Nullable>enable</Nullable>
        <RuntimeIdentifier>win-x86</RuntimeIdentifier>
    </PropertyGroup>

</Project>
```

and the csproj for the library code now:

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>net7.0</TargetFramework>
    <RuntimeIdentifier>win-x86</RuntimeIdentifier>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="MathNet.Numerics" Version="5.0.0" />
    <PackageReference Include="MKL.NET" Version="1.4.0" />
    <PackageReference Include="MKL.NET.Matrix" Version="1.0.0" />
    <PackageReference Include="MKL.NET.win-x86" Version="2022.0.0.115" />
  </ItemGroup>

  <PropertyGroup>
    <DebugType Condition=" '$(Configuration)' == 'Release' ">None</DebugType>
  </PropertyGroup>

</Project>
```

**Comment by AnthonyLloyd** [comment]
I've found the issue. It was missing from the package in 1.4.0 for win-x64. There was a strange issue that appeared due to them being also listed as content in the project and there being a conflict with the win-x64 name I think.
Thanks for pointing this out. Fixed in 1.4.1.

**Comment by alexbird1985** [comment]
I've tested it today and it is in fact completely resolved.
Using mkl for my various fft calls is about 2x speedup it seems so thanks for writing this library!
